client/components/jetpack/intro-pricing-banner/index.tsx

```diff
@@ -110,17 +111,19 @@ const IntroPricingBanner: React.FC = () => {
 					</div>
 					<div className="intro-pricing-banner__item is-agencies">
 						<img className="intro-pricing-banner__item-icon" src={ people } alt="" />
-						<button
+						<ExternalLinkWithTracking
 							className="intro-pricing-banner__item-label is-link"
-							onClick={ () => {
-								recordTracksEvent( 'calypso_jpcom_agencies_page_intro_banner_link_click' );
-								window.location.assign(
-									localizeUrl( 'https://cloud.jetpack.com/manage/pricing', locale )
-								);
-							} }
+							tracksEventName="calypso_jpcom_agencies_page_intro_banner_link_click"
```

Comment:
I tried looking at other tracks for the jetpack pricing page because this name didn't seem super clear to me what it actually is... I think this is probably fine, I had something like `calypso_jetpack_pricing_jpcom_agencies_page_intro_banner_link_click` but that is very long 😅 . I am unsure about this, let me know what you think 😄

Reply:
Hmm, it's an already existing tracks event with history of collected data. I hesitate to change the name because I'm not sure we want to break up the tracking data? 🤔

Reply:
My apologies, I didn't realize it was existing but I now see that very clearly in the code, apologies for that brain hiccup 😅